src/ansible_navigator/ui_framework/ui.py

```diff
@@ -299,16 +299,11 @@ def _scroll_bar(
     ) -> None:
         """Add a scroll bar if the lines > viewport
```

Comment:
FWIW this could be rewritten for humans. It seems to refer to (outdated?) identifiers and doesn't have a trailing period.

Reply:
I think it's better